cyclestreets.app/src/main/assets/whatsnew.html

```diff
@@ -15,7 +15,7 @@ <h3>Release 3.7 (BETA)</h3>
     <li>Modernisation of some icons, using vector graphics for better resolution and smaller install size.</li>
     <li>LiveRide now requests audio focus which should improve the experience e.g. when also playing music.</li>
     <li>Target streets are now described as part of instructions - e.g. "Get ready to turn right" becomes "Get ready to turn right onto [street name]".</li>
-    <li>Pressing "back" now takes you back to your previous view, so you don't accidentally minimise the app so often.</li>
+    <li>Revised Back button behaviour.</li>
```

Comment:
You should update `beta.txt` in line with these changes, so that the release notes on the play store get auto-updated when the new build is published.

Reply:
Revised -> Improved, I would suggest!

Reply:
Missed that - change made.